spec/migrations/20190118204247_move_miq_set_memberships_spec.rb

```diff
@@ -0,0 +1,53 @@
+require_migration
+
+describe MoveMiqSetMemberships do
+  let(:rel_stub) { migration_stub(:Relationship) }
+  let(:membership_stub) { migration_stub(:MiqSetMembership) }
+  let(:ns_stub) { Class.new(ActiveRecord::Base) { self.table_name = "miq_ae_namespaces" } }
+
+  migration_context :up do
+    context "two trivial trees" do
+      it 'creates set memberships' do
```

Comment:
business statement (not a question about the code)

What is this code doing?

It seems you should have a create_set and create_set_member helper.

As far as people are concerned, they have a set and members - not children and relationships and stuff.
Not saying the abstraction won't be a little leaky, but this is a tad bit confusing